airflow/docker-compose.yaml

```diff
@@ -81,10 +81,11 @@ x-airflow-common:
 
     CALITP_BUCKET__AIRTABLE: "gs://test-calitp-airtable"
     CALITP_BUCKET__DBT_ARTIFACTS: "gs://test-calitp-dbt-artifacts"
+    CALITP_BUCKET__GTFS_CONFIG: "gs://test-calitp-gtfs-download-config"
     CALITP_BUCKET__GTFS_RT_RAW: "gs://test-calitp-gtfs-rt-raw"
```

Comment:
should this also be updated?

Reply:
the RT raw?

Reply:
I thought we were moving to a v2 bucket even with just the metadata updates to make the backfill easier

Reply:
Ah yes I just wasn't sure what line you were referring to.

Reply:
ope yeah sorry I see now it's not the right one